tests/e2e/registry/test_registry_by_increasing_num_of_image_registry_pods.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+import logging
+
+from ocs_ci.ocs.constants import OPENSHIFT_IMAGE_REGISTRY_NAMESPACE
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.ocs.registry import (
+    validate_registry_pod_status,
+    image_pull, image_push, image_list_all, image_rm,
+    check_image_exists_in_registry, validate_pvc_mount_on_registry_pod,
+    modify_registry_pod_count
+)
+from ocs_ci.framework.testlib import E2ETest, workloads
+
+log = logging.getLogger(__name__)
+IMAGE_URL = 'docker.io/library/busybox'
+
+
+@workloads
+class TestRegistryByIncreasingNumPods(E2ETest):
+    """
+    Test to increase number of registry pods
+    and validate the registry pod increased
+    """
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request):
+        """
+        Clean up svt
```

Comment:
You are not using svt.  Please update the comment accordingly

Reply:
done